File: wlxx-services/wlxx-system/wlxx-system-client/src/main/java/com/xzg/wlxx/system/client/entity/po/OrgPo.java
package com.xzg.wlxx.system.client.entity.po;

import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableName;
import com.xzg.wlxx.common.base.BasePo;
import io.swagger.v3.oas.annotations.media.Schema;
import jakarta.validation.constraints.NotBlank;
import lombok.Data;
import lombok.EqualsAndHashCode;

@TableName("org")
@Data
@EqualsAndHashCode(callSuper = true)
public class OrgPo extends BasePo<OrgPo> {

    @TableField("org_code")
    @NotBlank(message = "不能为空")
    private String orgCode;

    @TableField("org_name")
    @NotBlank(message = "不能为空")
    private String orgName;
    private Integer orgLevel;
    private Long orgId;
    private Integer orgType;

    @Schema(description = "是否启用", defaultValue = "1")
    private Boolean enabled;
}

File: wlxx-services/wlxx-system/wlxx-system-client/src/main/kotlin/com/xzg/wlxx/system/client/entity/po/OrgPo.kt
package com.xzg.wlxx.system.client.entity.po

import com.baomidou.mybatisplus.annotation.TableField
import com.baomidou.mybatisplus.annotation.TableName
import com.xzg.wlxx.common.base.BasePo
import lombok.Data
import lombok.EqualsAndHashCode

@TableName("org")
@Data
@EqualsAndHashCode(callSuper = true)
data class OrgPo(
    @TableField("org_code")
    var orgCode: String? = null,

    @TableField("org_name")
    var orgName: String? = null,
    var orgLevel: Int? = null,
    var orgId: Long? = null,
    var orgType: Int? = null,
) : BasePo<OrgPo?>()
